### PR TITLE
[DOC] Docs update to add installing perses from source section.

### PR DESCRIPTION
**Pull request by eschabell**
# Description

Docs update to add installing perses from source section.

# Screenshots

# Checklist

- [ ] Pull request has a descriptive title and context useful to a reviewer.
- [ ] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the
  following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `DOC`,`IGNORE`.
- [ ] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [ ] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [ ] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [ ] Visual tests are stable and unlikely to be flaky.
  See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests)
  and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues
  include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

File: docs/user-guides/installing-perses-from-source.md [hunk 1]
```diff
@@ -3,4 +3,109 @@
...
+>> build the perses cli
+CGO_ENABLED=0 GOARCH=arm64 GOOS=darwin go build -ldflags "-s -w -X github.com/prometheus/common/version.Version=0.41.1
+Perses server built successfully!
+```
+
+Note the last line stating that you have successfully built your own instance of Perses!
+
+## Starting the Perses server
+
+To start the server you need a minimal configuration file, so let's set up a new one for you. Apply the following to a
```
Comment (Nexucis) [inline docs/user-guides/installing-perses-from-source.md:53]: That's actually not true, Perses can be started without providing any configuration file.

I think you shouldn't give a configuration like that, there is a high chance it doesn't match what users are looking for.

Probably you should just point to the doc about the configuration. Each feature should require a proper documentation that will contain the proper configuration.
From my point of view it's ok to skip this section. The scope of this part is just to build Perses from source. Configuring Perses is something else.

File: docs/user-guides/installing-perses-from-source.md [hunk 2]
```diff
@@ -3,4 +3,109 @@
...
+
+The database section points to a new directory called `myperses/local_db` where all your visualization and dashboard
+project files will be kept. The provisioning sections defines where all the JSON files will be stored, in
+`myperses/data`. The schema section defines where Perses can find the definitions for all things used to create
+visualization and dashboard elements.
+
+Once you have the configuration file set up, you can now start your Perses instance:
+
+```shell
+./bin/perses -config [PATH_TO]/myconfig.yaml
```
Comment (Nexucis) [inline docs/user-guides/installing-perses-from-source.md:82]:
```suggestion
./bin/perses
```

File: docs/user-guides/installing-perses-from-source.md [hunk 3]
```diff
@@ -3,4 +3,109 @@
...
+## Downloading the source
+
+You can obtain the source code of the Perses project at the main product page in various compression formats or you can
+clone the git repository as shown here:
+
+```shell
+git clone https://github.com/perses/perses.git
+```
+
+Now just change into the Perses project root directory and build the project:
```
Comment (Nexucis) [inline docs/user-guides/installing-perses-from-source.md:23]:
```suggestion
Now move to the perses folder and build the project
```

File: docs/user-guides/installing-perses-from-source.md [hunk 4]
```diff
@@ -3,4 +3,109 @@
 This sections continues with installing using the source code, test the project, build the project, configure examples,
 and run the Perses dashboard project on your local machine.
 
-## Coming soon...
+## Building requirements
+
+To build the Perses project from source you need minimum versions of the following:
+
+- GO v1.18+
```
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user-guides/installing-perses-from-source.md]: Fixed and pushed in branch.

File: docs/user-guides/installing-perses-from-source.md [hunk 5]
```diff
@@ -3,4 +3,109 @@
...
+  Time:    28.59s
+
+>> compressing assets
+scripts/compress_assets.sh
+GOARCH=arm64 GOOS=darwin go generate ./internal/api
+>> build the perses api
+CGO_ENABLED=0 GOARCH=arm64 GOOS=darwin go build -ldflags "-s -w -X github.com/prometheus/common/version.Version=0.41.1
+>> build the perses cli
+CGO_ENABLED=0 GOARCH=arm64 GOOS=darwin go build -ldflags "-s -w -X github.com/prometheus/common/version.Version=0.41.1
+Perses server built successfully!
```
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user-guides/installing-perses-from-source.md:46]: Fixed in branch.

File: docs/user-guides/installing-perses-from-source.md (hunk 1 above)
Comment (eschabell) [reply docs/user-guides/installing-perses-from-source.md:53]: Fixed in branch.

File: docs/user-guides/installing-perses-from-source.md (hunk 2 above)
Comment (eschabell) [reply docs/user-guides/installing-perses-from-source.md:82]: Fixed in branch.

File: docs/user-guides/installing-perses-from-source.md (hunk 3 above)
Comment (eschabell) [reply docs/user-guides/installing-perses-from-source.md:23]: Fixed in branch.

**Review by Nexucis — APPROVED** [review]
nice, thanks !

**Pull request merged**